src/metabase/server/middleware/session.clj

```diff
@@ -234,28 +236,30 @@
 
 (defn do-with-current-user
   "Impl for `with-current-user`."
-  [{:keys [metabase-user-id is-superuser? user-locale]} thunk]
+  [{:keys [metabase-user-id is-superuser? user-locale settings]} thunk]
   (binding [*current-user-id*              metabase-user-id
             i18n/*user-locale*             user-locale
             *is-superuser?*                (boolean is-superuser?)
             *current-user*                 (delay (find-user metabase-user-id))
-            *current-user-permissions-set* (delay (some-> metabase-user-id user/permissions-set))]
+            *current-user-permissions-set* (delay (some-> metabase-user-id user/permissions-set))
+            *user-local-values*            (atom (or settings {}))]
     (thunk)))
 
 (defmacro ^:private with-current-user-for-request
   [request & body]
   `(do-with-current-user ~request (fn [] ~@body)))
 
 (defn bind-current-user
-  "Middleware that binds `metabase.api.common/*current-user*`, `*current-user-id*`, `*is-superuser?*`, and
-  `*current-user-permissions-set*`.
+  "Middleware that binds `metabase.api.common/*current-user*`, `*current-user-id*`, `*is-superuser?*`,
+  `*current-user-permissions-set*, and `metabase.models.setting/*user-local-values*`.
```

Comment:
For next time, it's nice to make give stuff square bracket `[[...]]` references when you touch docstrings. I'm not sure if VIM fireplace supports it yet but CIDER and possibly a few others treat those as links so you can jump to the definition by clicking it or `M-.`

Reply:
Ah yeah, I forget because I can jump to definition in my vim setup without it. Will try to remember.